README.md

```diff
@@ -38,4 +38,7 @@ See [CONTRIBUTING](./CONTRIBUTING.md).
 
 # Sponsoring Services
 
+![GitHub-Mark-Light](/documents/resources/images/browserstack-darkmode.svg#gh-dark-mode-only)
+![GitHub-Mark-Dark](/documents/resources/images/browserstack-lightmode.svg#gh-light-mode-only)
```

Comment:
Let's use `<picture>` instead per GitHub's doc.

>  fragment appended to the URL (#gh-dark-mode-only or #gh-light-mode-only), is deprecated

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to

Reply:
fixed